tests/e2e/e2e_community_update_params_test.go

```diff
@@ -0,0 +1,168 @@
+		communitytypes.NewParams(
+			upgradeTime,
+			sdkmath.LegacyNewDec(1111), // stakingRewardsPerSecond
+			sdkmath.LegacyNewDec(2222), // upgradeTimeSetstakingRewardsPerSecond
+		),
+	)
+
+	// Make sure we're actually changing the params
+	suite.NotEqual(
+		updateParamsMsg.Params,
+		communityParamsResInitial.Params,
+		"new params should be different from existing",
+	)
+
+	proposalMsg, err := govv1.NewMsgSubmitProposal(
+		[]sdk.Msg{&updateParamsMsg},
+		govParamsRes.DepositParams.MinDeposit,
+		kavaAcc.SdkAddress.String(),
+		"community-update-params",
+	)
+	suite.NoError(err)
+
+	req := util.KavaMsgRequest{
+		Msgs:      []sdk.Msg{proposalMsg},
+		GasLimit:  uint64(gasLimit),
+		FeeAmount: sdk.NewCoins(fee),
+		Memo:      "this is a proposal please accept me",
+	}
+	res := kavaAcc.SignAndBroadcastKavaTx(req)
+	suite.Require().NoError(res.Err)
+
+	// Wait for proposal to be submitted
+	txRes, err := util.WaitForSdkTxCommit(suite.Kava.Tx, res.Result.TxHash, 6*time.Second)
+	suite.Require().NoError(err)
+
+	// Parse tx response to get proposal id
+	var govRes govv1.MsgSubmitProposalResponse
+	suite.decodeTxMsgResponse(txRes, &govRes)
+
+	// 2. Vote for proposal from whale account
+	whale := suite.Kava.GetAccount(testutil.FundedAccountName)
+	voteMsg := govv1.NewMsgVote(
+		whale.SdkAddress,
+		govRes.ProposalId,
+		govv1.OptionYes,
+		"",
+	)
+
+	voteReq := util.KavaMsgRequest{
+		Msgs:      []sdk.Msg{voteMsg},
+		GasLimit:  uint64(gasLimit),
+		FeeAmount: sdk.NewCoins(fee),
+		Memo:      "voting",
+	}
+	voteRes := whale.SignAndBroadcastKavaTx(voteReq)
+	suite.Require().NoError(voteRes.Err)
+
+	_, err = util.WaitForSdkTxCommit(suite.Kava.Tx, voteRes.Result.TxHash, 6*time.Second)
+	suite.Require().NoError(err)
+
+	// 3. Wait until proposal passes
+	suite.Require().Eventually(func() bool {
+		proposalRes, err := suite.Kava.Gov.Proposal(context.Background(), &govv1.QueryProposalRequest{
+			ProposalId: govRes.ProposalId,
+		})
+		suite.NoError(err)
+
+		return proposalRes.Proposal.Status == govv1.StatusPassed
+	}, 60*time.Second, 1*time.Second)
+
+	// Check parameters are updated
+	communityParamsRes, err := suite.Kava.Community.Params(
+		context.Background(),
+		&communitytypes.QueryParamsRequest{},
+	)
+	suite.Require().NoError(err)
+
+	suite.Equal(updateParamsMsg.Params, communityParamsRes.Params)
+}
+
+func (suite *IntegrationTestSuite) decodeTxMsgResponse(txRes *sdk.TxResponse, ptr codec.ProtoMarshaler) {
+	// convert txRes.Data hex string to bytes
+	txResBytes, err := hex.DecodeString(txRes.Data)
+	suite.Require().NoError(err)
+
+	// Unmarshal data to TxMsgData
+	var txMsgData sdk.TxMsgData
+	suite.Kava.EncodingConfig.Marshaler.MustUnmarshal(txResBytes, &txMsgData)
+	suite.T().Logf("txData.MsgResponses: %v", txMsgData.MsgResponses)
+
+	// Parse MsgResponse
+	suite.Kava.EncodingConfig.Marshaler.MustUnmarshal(txMsgData.MsgResponses[0].Value, ptr)
+	suite.Require().NoError(err)
+}
```

Comment:
nit: might be nice to move this to a stand alone function in e2e/testutil or somewhere so it can be reused in other tests. Auction and committee uses return data.